src/test_l1_plat_power.c

```diff
@@ -74,6 +74,30 @@
 static int gTestGroup = 1;
 static int gTestID = 1;
 
+#define PLAT_ASSERT_AUTO_TERM_NUMERICAL(value, comparison){\
+      if(value != comparison){\
+	        UT_LOG("\n In %s Comparison: [%d = %d]\n", __FUNCTION__, value, comparison);\
+		PLAT_TERM();\
+		UT_FAIL_FATAL();\
+      }\
+}\
+
+#define PLAT_ASSERT_AUTO_TERM_NOT_EQUAL(value, comparison){\
+	if(value == comparison){\
+		UT_LOG("\n In %s Comparison: [%d = %d]\n", __FUNCTION__, value, comparison);\
+		PLAT_TERM();\
+		UT_FAIL_FATAL();\
+	}\
+}\
+
+#define DS_ASSERT_AUTO_TERM_STRING(value, comparison){\
```

Comment:
Your aware right this one is misleading.

DS_ASSERT_AUTO_TERM_STRING - there's no String involved. 

PLAT_ASSERT_AUTO_TERM_NUMERICAL

You would be better calling the function `PLAT_ASSERT_WITH_TERM` then you don't need `STRING` or `NUMERICAL`

If your going to compare strings, you'll need to use `strcmp` or `strncmp` ideally.

Reply:
will remove those unnecessary definitions.

Reply:
The above-mentioned macros are now removed.  Thanks.